tiup/tiup-bench.md

```diff
@@ -152,8 +152,16 @@ Flags:
     tiup bench tpch --sf=1 prepare
     ```
 
-2. Run the TPC-H test by executing one of the following commands:
+2. Collect statistics:
+    For the OLAP scenarios, to ensure that the TiDB optimizer can generate the optimal execution plan, execute the following SQL statements to collect statistics in advance. **We need to set [tidb_analyze_column_options](/system-variables.md#tidb_analyze_column_options-new-in-v830) to `ALL`, otherwise it will result in a significant drop in query performance.**
```

Comment:
```suggestion

    For the OLAP scenarios, to ensure that the TiDB optimizer can generate the optimal execution plan, execute the following SQL statements to collect statistics in advance. **We need to set [tidb_analyze_column_options](/system-variables.md#tidb_analyze_column_options-new-in-v830) to `ALL`, otherwise it will result in a significant drop in query performance.**
```